tools/accuracy_checker/accuracy_checker/data_analyzer/coco_instance_segmentation_analyzer.py

```diff
@@ -0,0 +1,73 @@
+"""
+Copyright (c) 2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from collections import Counter
+from .base_data_analyzer import BaseDataAnalyzer
+from ..logging import print_info
+
+
+class CoCoInstanceSegmentationDataAnalyzer(BaseDataAnalyzer):
+    __provider__ = 'CoCoInstanceSegmentationAnnotation'
+
+    def analyze(self, result: list, meta, count_objects=True):
+        data_analysis = {}
+        if count_objects:
+            data_analysis['annotations_size'] = self.object_count(result)
+
+        counter = Counter()
+        total_instances = 0
+        characteristics = {}
+        for data in result:
+            total_instances += data.size
+            counter.update(data.labels)
+            for label, rect in zip(data.labels, data.metadata['rects']):
+                if characteristics.get(label):
+                    characteristics[label]['area'].append(float(rect[2] * rect[3]))
```

Comment:
area is not always equal area of box, please use `data.area` for getting correct area of mask